apps/docs/content/guides/database/connection-management.mdx

```diff
@@ -0,0 +1,98 @@
+---
+
+## Connections
+
+Every [Compute Add-On](https://supabase.com/docs/guides/platform/compute-add-ons) has a pre-configured direct connection count and Supavisor pool size. This guide discusses ways to observe and manage them resourcefully.
+
+## What is a pool
+
+When clients (backend servers) connect to PostgreSQL or Supavisor, they use the PostgreSQL Wire Protocol. Unlike HTTP requests, these connections are stateful and persistent, involving a costly startup but becoming efficient once established.
+
+Clients may create several active connections with Postgres, reducing the overhead of establishing connections between queries.
+
+## The pools in Supabase
+
+Supabase servers only create connections when needed, closing idle ones to conserve resources. If you're not using a service, no connections will be made. They are configured to not over-consume the database connections.
+
+## Understanding Supavisor
+
+Supavisor is used to connect with external libraries, especially for serverless deployments. In transaction mode, it acts as a database connection load balancer between your application and Postgres. This setup allows your clients (application servers) to share database connections more efficiently.
+
+In serverless or auto-scaling environments, Supavisor helps manage a large number of client connections by optimizing their use of available database connections.
+
+While Supavisor can handle an impressive amount of client connections, they are ultimately routed through a smaller pool of actual database connections, based on the capacity of your Postgres database.
+
+### Configuring Supavisor's pool size
+
+You can change how many database connections Supavisor can manage by altering the pool size in the [Database Settings](https://supabase.com/dashboard/project/_/settings/database):
+
+![Connection Info and Certificate.](/docs/img/database/pool-size.png)
+
+The general rule is that if you are heavily using the PostgREST database API, you should be conscientious about raising your pool size past 40%. Otherwise, you can commit 80% to the pool. This leaves adequate room for the Authentication server and other utilities.
+
+These numbers are generalizations and assume a certain level of activity from all connected servers. The actual values depend on your peak connection usage. For instance, if you were only using 80 connections in a week period and your database could support 500 connections, then realistically you could allocate the remaining 420 (minus a reasonable buffer) to service more demand.
+
+## Monitoring connections
+
+### Capturing historical usage
+
+Supabase offers a Grafana Dashboard that records and visualizes over 200 project metrics, including connections. For setup instructions, check the [metrics docs](/docs/guides/platform/metrics).
+
+Its "Client Connections" graph displays connections for both Supavisor and Postgres
+![client connection graph](/docs/img/database/client-connections.png)
```

Comment:
Image here also 404s

Reply:
Hmm, not really sure how I left these out. They should be present now